### ISSUE TITLE
Downloading notebook as PDF does not work

### ISSUE THREAD

**Issue by interrogator** [uzh/marugoto#23]
I tried to download the PDF contents of my Notebook. A download does start, for `marugoto-notebook.pdf`, but the content is not PDF:

```
<!DOCTYPE html><html lang=en><head><meta charset=utf-8><meta http
equiv=X-UA-Compatible content="IE=edge"><meta name=viewport
content="width=device-width,initial-scale=1"><link rel=icon href=
favicon.ico><title>Marugoto</title><link href="https://fonts.googleapis.com/css
family=Libre+Baskerville:400,400i,700&amp;subset=latin-ext" rel=stylesheet><link
href="https://fonts.googleapis.com/css?family=Kalam
300,400&display=swap&subset=latin-ext" rel=stylesheet><link href="https:/
fonts.googleapis.com/css?family=DM+Sans:400,500,700&display=swap"
rel=stylesheet><link href=/css/app.b3f83eb2.css rel=preload as=style><link href=
css/chunk-vendors.d4504d4a.css rel=preload as=style><link href=/js/app
8bd54bd3.js rel=preload as=script><link href=/js/chunk-vendors.959f57b0.js
rel=preload as=script><link href=/css/chunk-vendors.d4504d4a.css
rel=stylesheet><link href=/css/app.b3f83eb2.css rel=stylesheet><
head><body><noscript><strong>We're sorry but hello-world doesn't work properly
without JavaScript enabled. Please enable it to continue.</strong></noscript><div
id=app></div><script src=/js/chunk-vendors.959f57b0.js></script><script src=/js
app.8bd54bd3.js></script></body></html
```

**Comment by christian-bernet** [comment]
Estimate: 1 hour

**Comment by martindusinberre** [comment]
I have closed this becaus it's also #38.
